tests/unit/cluster/hostnames.tcl

```diff
@@ -116,10 +116,11 @@ test "Verify the nodes configured with prefer hostname only show hostname for ne
     # Have everyone forget node 6 and isolate it from the cluster.
     isolate_node 6
 
-    # Set hostnames for the masters, now that the node is isolated
-    R 0 config set cluster-announce-hostname "shard-1.com"
-    R 1 config set cluster-announce-hostname "shard-2.com"
-    R 2 config set cluster-announce-hostname "shard-3.com"
+    set primaries 3
+    for {set j 0} {$j < $primaries} {incr j} {
```

Comment:
Super nit - starting `j` from `1` would avoid the changes on L154 and L160. It took me a while to understand the shift of the shard "index". 
```suggestion
    for {set j 1} {$j < $primaries} {incr j} {
```

Reply:
The server nodes are 0-indexed so we would have to perform an expression to apply the 1-based shard id appropriately. Hence, I think the current change is more simpler.  

0th node -> shard-0.com
1st node -> shard-1.com and so on.